### PR TITLE
Add Category REST endpoint

### PR DESCRIPTION
**Pull request by jayachristina**

### CODE REVIEW

File: src/main/java/org/globex/retail/store/catalog/model/entity/Category.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
 @Entity(name = "Category")
 @Table(name = "category")
 @Cacheable
+
+
```
Comment (btison) [inline src/main/java/org/globex/retail/store/catalog/model/entity/Category.java]: can you delete these two blank lines?

File: src/main/java/org/globex/retail/store/catalog/model/entity/Category.java [hunk 2]
```diff
@@ -15,5 +17,5 @@ public class Category extends PanacheEntityBase {
 
     @Column(name = "category")
     public String category;
-
+
```
Comment (btison) [inline src/main/java/org/globex/retail/store/catalog/model/entity/Category.java]: Can you delete the whitespace?

File: src/main/java/org/globex/retail/store/catalog/service/CatalogService.java [hunk 3]
```diff
@@ -100,4 +101,11 @@ private List<ProductDto> toDto(List<Product> productList) {
         return productList.stream().map(ProductMapper::toDto).collect(Collectors.toList());
     }
 
+    
+    public List<Category> getCategoryList() {
+        List<Category> categoryList = Category.listAll();
+        
+        return categoryList;
+    }
+
```
Comment (btison) [inline src/main/java/org/globex/retail/store/catalog/service/CatalogService.java:108]: you could simplify this to `return Category.listAll();`

**Review by btison — CHANGES_REQUESTED** [review]

File: src/main/java/org/globex/retail/store/catalog/model/entity/Category.java (hunk 1 above)
Comment (jayachristina) [reply src/main/java/org/globex/retail/store/catalog/model/entity/Category.java]: Done thanks!

File: src/main/java/org/globex/retail/store/catalog/model/entity/Category.java (hunk 2 above)
Comment (jayachristina) [reply src/main/java/org/globex/retail/store/catalog/model/entity/Category.java]: Done

File: src/main/java/org/globex/retail/store/catalog/service/CatalogService.java (hunk 3 above)
Comment (jayachristina) [reply src/main/java/org/globex/retail/store/catalog/service/CatalogService.java:108]: Done

**Review by btison — APPROVED** [review]

**Pull request merged**